schemas/2016-09-01-preview/Microsoft.Solutions.json

```diff
@@ -0,0 +1,241 @@
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The appliance properties."
+        },
+        "plan": {
+          "oneOf": [
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/resorucePlan"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The plan information."
+        },
+        "kind": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[-\\w\\._,\\(\\)]+$"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The kind of the appliance. Allowed values are MarketPlace and ServiceCatalog."
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "kind"
+      ],
+      "description": "Microsoft.Solutions/appliances"
+    }
+  },
+  "definitions": {
+    "ApplianceArtifact": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The appliance artifact name."
+        },
+        "uri": {
+          "type": "string",
+          "description": "The appliance artifact blob uri."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Template",
+            "Custom"
+          ],
+          "description": "The the appliance artifact type."
+        }
+      },
+      "description": "Appliance artifact."
+    },
+    "ApplianceDefinitionProperties": {
+      "type": "object",
+      "properties": {
+        "lockLevel": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "CanNotDelete",
+                "ReadOnly",
+                "None"
+              ]
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The appliance lock level."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The appliance definition display name."
+        },
+        "authorizations": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ApplianceProviderAuthorization"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
```

Comment:
It's possible... You can have a parameter with type="array". 

---
In reply to: [115861471](https://github.com/Azure/azure-resource-manager-schemas/pull/316#discussion_r115861471) [](ancestors = 115861471)